content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,232 @@
 ====================
-After Sales Features
+After-Sales services
 ====================
 
-As your business grows, having the right tool to support your helpdesk team on recording, tracking
-and managing issues raised easy and efficiently, is key. Odoo’s Helpdesk application allows you to
-generate credit notes, manage returns, products, repairs, grant coupons, and even plan onsite
-interventions from a ticket’s page.
+*After-Sales* services can be configured in the *Helpdesk* application for individual *teams*. Once
+enabled, users can issue refunds, process returns, generate coupons, and/or schedule repair and
+field service interventions directly from a ticket.
 
-Set up the after sales services
+Set up the after-sales services
 ===============================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable the after sales
-options: *Refunds, Returns, Coupons, Repairs and Onsite Interventions*.
+Go to :menuselection:`Helpdesk --> Configuration --> Teams` to select a :guilabel:`Team`. Scroll to
+the :guilabel:`After-Sales` section and choose which of the following options to enable:
```

Comment:
I capitalized 'After-Sales' in the first sentence "Start by enabling After-Sales services..." for consistencies sake (that is how it is on the UI and in the rest of the doc), but otherwise I am ok with this change 👍